tests/test_input.py

```diff
@@ -0,0 +1,38 @@
+from pydicer.input.web import WebInput, download_and_extract_zip_file
+from pydicer.input.test import TestInput
+from pydicer.input.filesystem import FilesystemInput
+
+import os
+
+def test_input_valid_working_dir_():
+    valid_test_input = WebInput(data_url="")
+    # Assert path to DICOMs exists
+    assert os.path.isdir(valid_test_input.working_directory)
```

Comment:
Since I think we said we would make sure all path's are stored in pathlib's Path objects, here we can just do (no need for os import):

`assert valid_test_input.working_directory.is_dir()`

Reply:
I commented it out for when we want to run tests locally and not have to download the data every time. Happy to redirect this particular test to a different directory and have vsc not run it in pytest extension or just uncomment it completely